src/wrappers/themis/android/build.gradle

```diff
@@ -0,0 +1,108 @@
+apply plugin: 'com.android.library'
+
+buildscript {
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.2.1'
```

Comment:
I thought, we moved to gradle 5?

Reply:
Uh... To be honest, I'm not sure how this affects the build. Maybe it tells Gradle to use this compatibility version? This line has been like this before the file got moved, when we were using Gradle 4.6. Running `./gradlew wrapper --gradle-version 5.6` upgraded only wrapper, it did not touch this line.

I guess it's worth looking into. Maybe we're using Gradle 5.6 to be able to run with Java 11, but the build still uses 3.2.1 mode. I'm not sure we need to upgrade that if it works fine, but I guess I can try and see what happens. Maybe we don't even need this line at all.

Reply:
Neither I know what this line indicates.. Maybe minimum gradle version?

If Android studio / IDEA doesn’t warn about this, we can leave it as is?

Reply:
I guess that's a different thing after all. There is Gradle as a command-line build tool (we use 5.6) and there is Gradle *library* with various runtime thingies. That line specifies the latter.

The project opens fine in Android Studio:

<img width="287" alt="Themis in Android Studio" src="https://user-images.githubusercontent.com/1256587/80805234-2b05bf00-8bc0-11ea-87ae-8868454e5078.png">

It builds successfully, runs tests, ~~eats 5+ GB of disk space for SDK & NDK,~~ etc.

Android Studio suggests to update the version to 3.6.3 which is the latest stable one in [Google's Maven repo](https://maven.google.com/web/index.html), along with some other updates in Android unit test runner (1.1 → 1.2) and Gradle wrapper version (5.6 → 5.6.4). And for some reason requires an additional property after an update. The strategy “blindly apply whatever Android Studio suggests” has worked fine before 🤷

The build and tests pass, so I guess let's update all of that too.

Reply:
> The strategy “blindly apply whatever Android Studio suggests” has worked fine before 🤷

I use the same, never failed for me :)

Reply:
Well, it does fail now.

It seems that 3.6.X has its issues, suggesting a downgrade to 3.5.X. However, 3.5.X has different issues. FFS, Android ecosystem :shrug: Since this upgrade is discretionary and we don't have any specific reason to upgrade other than being up-to-date, let's keep using 3.2.1 which worked fine before.

Reply:
Well, Android ecosystem ¯\_(ツ)_/¯